randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1823,7 +1823,7 @@ Extra - asset_id: collision_camera_025
               Any of the following:
                   Walljump (Intermediate)
                   Walljump (Beginner) and Use Spin Boost
-          Bomb and Morph Ball and Infinite Bomb Jump (Advanced)
+          Infinite Bomb Jump (Advanced) and Lay Bomb
```

Comment:
Not doable with Cross Bombs, so leave it separated out.

Reply:
Done